resolwe_bio/tools/make_igv_session.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name,import-error
+"""Create IGV session."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+from lxml import etree
+
+parser = argparse.ArgumentParser(description='Create igv session')
+
+parser.add_argument('-bam', '--bam_path', nargs='+', default=[], help='List of paths to bam file')
```

Comment:
In the script all arguments are required. Genome id is required, to upload reference genome in IGV session together with bam sessions, and locus is by default set to All in yml. file. f I add metavar in arguments the xml file is not created in right way, that is why I set the required argument to True.